src/lib/aws/aws_common_resources.py

```diff
@@ -0,0 +1,6 @@
+
+
+class AWS_Common_Resources:
```

Comment:
The class should be named in CamelCase manner, without underscores.

Reply:
fixed